test/hubspot/main.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/amp-labs/connectors"
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/hubspot"
+	"github.com/amp-labs/connectors/test/utils"
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/spyzhov/ajson"
+	"golang.org/x/oauth2"
+)
+
+// Contact is a basic Hubspot contact.
+type Contact struct {
+	Email     string `json:"email"`
+	Phone     string `json:"phone"`
+	Company   string `json:"company"`
+	Website   string `json:"website"`
+	Lastname  string `json:"lastname"`
+	Firstname string `json:"firstname"`
+}
+
+func main() {
```

Comment:
Can you please add comments on how to run this test, and also create a sample creds file in OnePassword so others on the team can easily get up and running with this test?